searx/engines/wordnik.py

```diff
@@ -0,0 +1,129 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+ Wordnik (general)
+"""
+
+import re
+from lxml.html import fromstring
+from searx.utils import extract_text
+from searx.raise_for_httperror import raise_for_httperror
+
+# about
+about = {
+    "website": 'https://www.wordnik.com',
+    "wikidata_id": 'Q8034401',
+    "official_api_documentation": None,
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": 'HTML',
+}
+
+categories = ['general']
+paging = False
+
+URL = 'https://www.wordnik.com'
+SEARCH_URL = URL + '/words/{query}'
+
+
+def request(query, params):
+    params['url'] = SEARCH_URL.format(query=query)
+    return params
+
+
+def response(resp):
+    if resp.status_code == 404:
```

Comment:
I think HTTP 404 is handled by the online plugin, there is no need for this check.

Reply:
OK.